corpus/tests.txt

```diff
@@ -147,8 +147,8 @@ Curly braces in description don't conflict w/ inline tags
 @param w/ named type
 =====================
 /**
-* @param MyClass $foo
-*/
+ * @param MyClass $foo
+ */
```

Comment:
Good catch. I had totally missed that the leading whitespace was missing from these! 

I have to admit, though, that it might be nice to have at least 1 test that makes sure that things still work OK even w/o leaving whitespace. Maybe it should just be a separate test, though.

Reply:
Agree that we should start to split tests up a bit. I've already added some more permutations but they are far from complete. In my experience it doesn't hurt to have extensive tests to cover at least all the most common combinations.

But that's something for another PR.